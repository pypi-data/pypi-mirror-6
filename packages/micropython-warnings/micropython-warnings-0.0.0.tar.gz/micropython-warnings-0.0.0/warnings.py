# MicroPython doesn't support warnings
