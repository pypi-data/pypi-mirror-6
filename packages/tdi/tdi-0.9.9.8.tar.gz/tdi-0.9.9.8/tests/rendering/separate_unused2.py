#!/usr/bin/env python
import warnings as _warnings
_warnings.resetwarnings()
_warnings.filterwarnings('error')

import warnings

from tdi import NodeWarning
from tdi import html

warnings.filterwarnings("error", category=NodeWarning)

try:
    template = html.from_string("""
    <node tdi="item">
        <node tdi="nested">
            <node tdi="subnested"></node>
        </node><tdi tdi=":-nested">
        </tdi><tdi tdi=":-nested2">
        </tdi><tdi tdi=":-nested3">
        </tdi>
    </node>
    """.lstrip())
except NodeWarning, e:
    print "OK: Nodewarning: %s" % (str(e),)
