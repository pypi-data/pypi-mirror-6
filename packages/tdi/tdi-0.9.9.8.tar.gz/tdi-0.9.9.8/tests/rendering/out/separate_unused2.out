OK: Nodewarning: Ignoring separator node(s) without accompanying content node: 'nested2', 'nested3'
